Submit Final Project proposal

h2doan opened the pull request.

h2doan committed:
Submit Final Project proposal

jamdon commented:
For the actual e-commerce part of your project (the cart and checking out portions) you may want to find some plugins that do part of this for you as that could get fairly complex quickly.

h2doan committed:
submit final project

h2doan commented:
Hi James,
I'm sorry I had lots of problems implementing my original final project proposal.  So, I changed your food-truck-app a bit to have slightly different data and added a Payment Types view.
Regards,

jamdon commented:
👍
